apis/proto/v1beta1/fn/run_function.proto

```diff
@@ -0,0 +1,131 @@
+/*
+Copyright 2022 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+syntax = "proto3";
```

Comment:
Can we setup ~linting and~ code generation here, similar to what we have in c/c?

https://github.com/crossplane/crossplane/blob/59b2e12f/apis/generate.go#L81

Linting via `buf lint` would be nice too, but we can add that later.

Reply:
We do have code generation already in place [here](https://github.com/crossplane/crossplane-runtime/blob/bbb004932c1e5eb472f013245f2614d9281d7af7/apis/apis.go#L38), and I used `make generate` to generate the code from the `.proto` file directly. I can add linting though!

Reply:
Ah I see that's a to do in crossplane too, I'll see if I can find my way through the build submodule 😆